src/main/cpp/t06_longest.cpp

```diff
@@ -9,18 +9,39 @@
 //Выведите самое длинное слово. Если таких слов несколько, то выводить нужно, которое встречается раньше.
 //
 //Sample Input:
-//
+//        8  11 14
 //Everyone of us has all we need
 //Sample Output:
 //
 //Everyone
 
 #include "t06_longest.h"
 #include <iostream>
+#include <string>
 
 
 using namespace std;
 
 int t06_longest() {
+    string s;
+    getline(cin, s);
+    int end, max, beg, maxbeg, n, maxend;
+    n = s.length();
+    maxend = s.find (' ');
+    max = maxend;
+    beg = maxend + 1;
+    maxbeg = 0;
 
+    for (end = s.find (' ', beg); end != -1; end = s.find (' ', beg) ) {
+        if (end - beg > max) {
+            max = end - beg; maxbeg = beg; maxend = end;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'maxend' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4486782)